web/crux-ui/e2e/image-config.spec.ts

```diff
@@ -0,0 +1,177 @@
+import { expect, Page, test } from '@playwright/test'
+import { imageConfigUrl } from './../src/routes'
+import { screenshotPath } from './utils/common'
+import { createImage, createProduct, createVersion } from './utils/products'
+
+const setup = async (
+  page: Page,
+  productName: string,
+  versionName: string,
+  imageName: string,
+): Promise<{ productId: string; versionId: string; imageId: string }> => {
+  const productId = await createProduct(page, productName, 'Complex')
+  const versionId = await createVersion(page, productId, versionName, 'Incremental')
+  const imageId = await createImage(page, productId, versionId, imageName)
+
+  return {
+    productId,
+    versionId,
+    imageId,
+  }
+}
+
+test.use({
+  viewport: { width: 1920, height: 1080 },
```

Comment:
I assume we'll always using this viewport, can we outsource this in a test const?